apps/api-gateway/src/authz/jwt-issuer.util.spec.ts

```
@@ -0,0 +1,31 @@
+import { getTrustedJwksUri, getTrustedJwtIssuers } from './jwt-issuer.util';
+
+describe('trusted JWT issuers', () => {
+  it('derives the Keycloak realm issuer when an explicit allowlist is not configured', () => {
+    expect(getTrustedJwtIssuers({ KEYCLOAK_DOMAIN: 'https://identity.example/', KEYCLOAK_REALM: 'platform' })).toEqual([
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)